Specialized fields Jigsaw (#2)
Ada-Developers-Academy/core-interlude-2: career-paths/activity-specialized-fields-jigsaw.md

# Activity: Specialized Fields Jigsaw

## Goal

Our goal is to learn and teach each other about different specialized fields within computing.

We should come out of this activity seeing the breadth and diversity of different specialized fields in computing. This activity should inspire us and show us just a few different options of career development.

## Preparation and Format

In this activity, we will be learning and then teaching each other. We will have two rounds: the research round and the teaching round.

> In the research round, you will meet with your first assigned group to learn about an assigned topic.
> 
> In the teaching round, your research round group will disseminate and you will go into your second assigned group. It is your responsibility to teach this group about that topic.

Reviewer: "assigned" twice in such close proximity sounds redundant
"disseminate" is transitive (it needs a direct object to denote what is being spread (esp. information)
more typical to "get into" groups
```suggestion
In the research round, you will meet with your first assigned group to learn about a specified topic.

In the teaching round, your research round group will disband and you will get into your second assigned group. It is your responsibility to teach this group about that topic.
```

Small groups should be prepared. In the teaching round, small groups should consist of one person from each research group.

### Example

Research Round groups:

| Research Group A | Research Group B | Research Group C |
| ---------------- | ---------------- | ---------------- |
| Olivia           | Corinne          | Briana           |
| Annie            | India            | Jayden           |

Teaching Round groups:

| Teaching Group A | Teaching Group B |
| ---------------- | ---------------- |
| Olivia           | Annie            |
| Corinne          | India            |
| Briana           | Jayden           |

## Activity Instructions

Spend 30 minutes with your research group in the research round. Then, spend the rest of the time in your teaching groups. Each person will be responsible for teaching the material to everyone else. Use any materials to teach the others. Using notes and visuals are great options.

### Research Round

> Each research group will be assigned an API to research.
> 
> You will be learning and teaching each other about other APIs, what they do, what they can't do, and how to read the documentation.

Reviewer: left over from API jigsaw?
```suggestion
Each research group will be assigned a specialized field to research.

You will be learning and teaching each other about other fields, what they focus on, how they are applied, and how to get started with them.
```

### Specialized Fields

1. Data Science, Data Analytics, Big Data
1. Artificial Intelligence, Machine Learning
1. Human-Computer Interaction (HCI), Computational Biology
1. Wearable Computing, Internet of Things
1. Augmented Reality/Virtual Reality
1. Computer Vision, Natural Language Processing

### !callout-info

## Multiple Related or Unrelated Fields

Each prompt lists at least two specialized fields. These fields _may_ be related to each other, and they may _not_ be.

### !end-callout

### Where to Start

For each field, answer these questions:

1. What is this field? What kind of problems does it try to solve?

> 1. What are some example projects, products, studies, or piece of work that is related to this field?

Reviewer: numerosity
```suggestion
1. What are some example projects, products, studies, or pieces of work that are related to this field?
```
